### PR TITLE
[chore] Fix linter issues

### PR DESCRIPTION
**Pull request by WawrzenskiPatryk**
All the changes are related to logs from running `yarn lint`.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -4033,7 +4033,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"eslint-import-resolver-typescript@npm:^3.5.2":
+"eslint-import-resolver-typescript@npm:3.5.3, eslint-import-resolver-typescript@npm:^3.5.2":
```
Comment (WawrzenskiPatryk) [inline yarn.lock:4036]: This is strange that the package was there in another version, yet without adding it to `package.json` we had problems like this:
image
image

File: package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     "generate": "nuxt generate",
     "preview": "nuxt preview",
     "postinstall": "nuxt prepare",
-    "lint": "eslint --ext .ts,.js,.vue .",
+    "lint": "eslint --ext .ts,.js,.vue . --fix",
```
[the remark this answers is not in the document]
Comment (elmir-ismayilzada) [reply package.json:9]: I recommend to add a new command "lint-fix"

File: yarn.lock (hunk 1 above)
Comment (elmir-ismayilzada) [reply yarn.lock:4036]: That's strange, really

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (WawrzenskiPatryk) [reply package.json]: Done ✅

File: components/ProductsList.vue [hunk 3]
```diff
@@ -34,7 +34,7 @@
 
 <script setup lang="ts">
 import { FullProduct } from '~/types/products';
-import ArrowIcon from '@/assets/icons/arrow_thin_left.svg?skipsvgo';
+// import ArrowIcon from '@/assets/icons/arrow_thin_left.svg?skipsvgo';
```
Comment (zakrzaq) [inline components/ProductsList.vue:37]: This line can be safely removed.

As well a lines [5-12] if you are there, pretty please

**Review by zakrzaq — APPROVED** [review]
👍🏻

File: components/ProductsList.vue (hunk 3 above)
Comment (WawrzenskiPatryk) [reply components/ProductsList.vue]: Done ✅

**Review by elmir-ismayilzada — APPROVED** [review]
Looks good 👍

**Review by zakrzaq — APPROVED** [review]

**Pull request merged**